apps/web/src/server/applications/case/project-team/applications-project-team.service.js

```diff
@@ -18,9 +18,13 @@ export const searchProjectTeamMembers = async (searchTerm, allAzureUsers, pageNu
 	const searchResults = allAzureUsers.filter((azureUser) => {
 		const { givenName, surname, userPrincipalName: email } = azureUser;
 
-		const recordString = `${givenName || ''} ${surname || ''} ${email || ''}`.toLocaleLowerCase();
-
-		return recordString.includes(searchTerm);
+		// check whether the search matches perfectly the name, the surname or the email
+		// for example, ‘Mil’ will not return ‘Miles’. Users will need to enter ‘Miles’
+		return (
+			`${givenName || ''}`.toLocaleLowerCase() === searchTerm ||
```

Comment:
reviewing the ticket I noticed this requirement I didn't see the first time.
Now only exact matches will produce results